lib/redi_search/lazily_load.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "forwardable"
```

Comment:
Could we move this to the top of `lib/redi_search.rb`? That way we'll make sure all files will have it when they need since we use it in other classes too. Could we also add `require "delegate"` while we are there too?

Reply:
Yep, sure it is right place for it. Sorry, I don’t understand why it needs to require "delegate", can you clarify please?

Reply:
No worries. When I was running `bin/console` it was showing SimpleDelegator couldnt be found. I'm wondering if something in 2.7 changed so it wasnt already required.